README.md

```diff
@@ -1 +1,143 @@
-# blog
```

Comment:
![image](https://github.com/Fahim-Khan-P/blog/assets/93039359/cf62debc-0597-4ea2-bdf7-a1023d4ff5d8)

**[OPTIONAL]**

- [ ] It would be nice if you make your commit message capitalized to make your pull request professional and descriptive 😊